GLUC-009 Recipe filtering

khristova21 opened the pull request.

fjcu256 commented:
Delete the Secrets.plist file, and then you can merge. After it is merged, then send the Secrets.plist file with the API ID and keys to the discord for us to all download and put in the correct location

khristova21 commented:
I removed `Secrets.plist` from the PR and sent the functional version to the team's Discord. It contains the id and key needed for Recipe API to run. Since it is in the .gitignore the file should be added after the initial merge of the recipe api implementation.

**To run and test, add `Secrets.plist` to `application/GlucoseGenie/`.** 

I also fixed the existing merge conflicts regarding the new file schemas in `GlucoseGenie.xcodeproj/project.pbxproj`.

fjcu256 reviewed (approved):
LGTM, just rename the commit to indicate that this is phase/part 1 of GLUC-013
